CHANGELOG.md

```diff
@@ -7,54 +7,60 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [1.7.0] - 2024-06-25
 ### Added
-- [GRD-797](https://jira.oicr.on.ca/browse/GRD-797) - add vidarr labels to outputs (changes to medata only)
+- [GRD-797](https://jira.oicr.on.ca/browse/GRD-797) - add vidarr labels to outputs (changes to medata only).
 
 ## [1.6.0] - 2024-02-02
```

Comment:
There are some tags missing, it may need additional work - at least all 3-digit tags need to be here with some information on what was done